### ISSUE TITLE
Adaptive card doesn't trigger topic in PVA

### ISSUE THREAD

**Issue by ryub3n** [microsoft/BotFramework-Composer#9277]
## Describe the bug
Adaptive card doesn't trigger topic in PVA when data value intent is set

## Version
Version: 2.1.2
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser

- [x] Electron distribution
- [x] Chrome
- [x] Safari
- [ ] Firefox
- [x] Edge

## OS

- [x] macOS
- [ ] Windows
- [ ] Ubuntu

## To Reproduce

1. Create a PVA bot and reuse the Enterprise Assistant Bot 'Bot Tour' dialog
2. It sends the initial card
3. Click on 'Calendar' or 'People' button.
4. The bot is not able to understand the intent in PVA.

## Expected behavior
The PVA should be able to handle the Action.Submit with 'intent' name in data and invoke the respective topic.

## Screenshots
This is the place where we inform what intent to be invoked. But it doesn't work for PVAs.
Also, where is the documentation for the data attribute the the supported values for Composer or PVAs?
<img width="606" alt="image" src="https://user-images.githubusercontent.com/24279903/177331397-02d15e9b-2c79-49b1-9cd9-531270cd11ce.png">

In PVA, even if I have a topic 'CalendarTour' it doesn't get invoked.
<img width="382" alt="image" src="https://user-images.githubusercontent.com/24279903/177332377-bcdb939e-2354-4a11-b719-ae8ac085b6c2.png">

**Comment by ramfattah** [comment]
Hey @ryub3n,

Could you please kindly share more details for step number 1, regarding how to `reuse` the 'Bot Tour' dialog?

> 1. Create a PVA bot and reuse the Enterprise Assistant Bot 'Bot Tour' dialog

**Comment by ramfattah** [comment]
@ryub3n to clarify, are you experiencing this issue specifically when using [BotTourDialog.dialog](https://github.com/microsoft/botframework-components/blob/main/generators/generator-bot-enterprise-assistant/generators/app/templates/dialogs/BotTourDialog/BotTourDialog.dialog) or just in general Adaptive Cards doesn't trigger topics in PVA?

**Comment by ryub3n** [comment]
@ramfattah , in general Adaptive Cards don't trigger topics in PVA.

**Comment by ramfattah** [comment]
Hi @ryub3n, we are working on this. We will report back once we have updates.

Thanks for your patience.

**Comment by ramfattah** [comment]
Hi @ryub3n.

I'm not able to reproduce this issue in PVA after clicking on the Adaptive card 'Calendar' or 'People' buttons.

#

Sharing steps I took:
1. Opened PVA bot in Composer
2. `+ Add a dialog`, named it BotTourDialog 
3. In the new dialog, I reused Enterprise [BotTourDialog.dialog](https://github.com/microsoft/botframework-components/blob/main/generators/generator-bot-enterprise-assistant/generators/app/templates/dialogs/BotTourDialog/BotTourDialog.dialog) line 1 - [95](https://github.com/microsoft/botframework-components/blob/a162d69a42b6ac81e2b59d4b3d89f7c864a75bc5/generators/generator-bot-enterprise-assistant/generators/app/templates/dialogs/BotTourDialog/BotTourDialog.dialog#L95)
4. In Composer, `Bot Responses -> BotTourDialog -> Show code` and added the LG code from [BotTourDialog.en-us.lg](https://github.com/microsoft/botframework-components/blob/main/generators/generator-bot-enterprise-assistant/generators/app/templates/dialogs/BotTourDialog/language-generation/en-us/BotTourDialog.en-us.lg) 
6. Then added the LG code from [cards.en-us.lg](https://github.com/microsoft/botframework-components/blob/main/generators/generator-bot-enterprise-assistant/generators/app/templates/language-generation/en-us/cards.en-us.lg) & [icons.en-us.lg](https://github.com/microsoft/botframework-components/blob/main/generators/generator-bot-enterprise-assistant/generators/app/templates/language-generation/en-us/icons.en-us.lg) into [BotTourDialog.en-us.lg](https://github.com/microsoft/botframework-components/blob/main/generators/generator-bot-enterprise-assistant/generators/app/templates/dialogs/BotTourDialog/language-generation/en-us/BotTourDialog.en-us.lg)
     * Attached is a zip containing final example of [BotTourDialog.en-us.zip](https://github.com/microsoft/BotFramework-Composer/files/9156704/BotTourDialog.en-us.zip) file
7. Published the bot in Composer
8. Tested the Adaptive Card in PVA


Screenshot
<img src="https://user-images.githubusercontent.com/38049078/180102080-492c6efe-f896-4308-9a23-942cb96599aa.png" width="500"/>


Hope this helps, let me know if you have any questions. Thanks.

**Comment by ryub3n** [comment]
@ramfattah, you're half way there. I meant that adaptive cards don't trigger **Topics**.

Continue with the below steps:

- Remove the PeopleTour and CalendarTour _intents_ in Composer and add them as **_topics_** in PVA.

> DELETE:
> <img width="113" alt="image" src="https://user-images.githubusercontent.com/24279903/180165196-be5259d7-89ec-4102-87b2-2da57d8f37e5.png">
> ADD IN PVA:
> <img width="732" alt="image" src="https://user-images.githubusercontent.com/24279903/180166111-9c6f3aad-ee2f-4a7a-a17b-406a2c8d2a27.png">


- The expectation is that a **PVA topic** can be triggered from an adaptive card as it's basically an intent.

**Comment by ramfattah** [comment]
@ryub3n, 

Thanks for providing the detailed steps.
I was able to reproduce this issue after adding the PeopleTour and CalendarTour as topics in PVA.

![image](https://user-images.githubusercontent.com/38049078/180183097-3ea9878f-cafb-41c0-8e8b-a1aceb716867.png)

**Comment by ramfattah** [comment]
@ryub3n,

Thanks for providing the detailed steps.
I was able to reproduce this issue after adding PeopleTour and CalendarTour as Topics in PVA.
![image](https://user-images.githubusercontent.com/38049078/180186742-e0af69ec-58a4-4331-ae9f-e1ea8250533b.png)

**Comment by ramfattah** [comment]
Hi @ryub3n,

Looks like triggering a topic with an adaptive card may not be supported in PVA. 

we found a workaround to redirect to a PVA topic without using adaptive cards. 
I was able to set up a dialog and redirect to PVA topic using "Begin a Power Virtual Agents Topic" action. [Documentation here](https://docs.microsoft.com/en-us/power-virtual-agents/advanced-bot-framework-composer#:~:text=Begin%20a%20Power%20Virtual%20Agents%20Topic)
